chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/configuration/AssaultProperties.java

```diff
@@ -46,6 +48,8 @@
 
   private boolean killApplicationActive = false;
 
+  private String killApplicationCronExpression;
```

Comment:
The new cronExpressions are missing a default value. Eg. "OFF"?

I saw that we don't consider null values, but it should align with the docs. Also the user should know how he can disable an expression (so I guess having the default value "OFF" is the better choice).